### DISCUSSION TITLE
想知道taro3是如何覆盖的vue主页面的render函数，应该看源码的哪里呀？

### DISCUSSION THREAD

**Discussion by alamhubb** [NervJS/taro#11778] [Q & A] [answered]
![9c3fde946ae9486a7e1c40ef092b1f1](https://user-images.githubusercontent.com/14007972/167258980-de854cca-b423-434f-a6dd-a49a4c82a17a.png)

想知道render被覆盖的逻辑是如何实现的

**Comment by b2nil** [comment, marked as answer]
https://github.com/NervJS/taro/blob/next/packages/taro-plugin-vue3/src/runtime/connect.ts#L134-L136
